modules/apps/data-engine/data-engine-rest-impl/src/main/java/com/liferay/data/engine/rest/internal/resource/v2_0/DataDefinitionResourceImpl.java

```diff
@@ -1221,6 +1225,56 @@ private void _removeFieldsFromDataListViews(
 		}
 	}
 
+	private void _reorderNestedDDMFormFields(DDMForm ddmForm) throws Exception {
+		for (DDMFormField ddmFormField : ddmForm.getDDMFormFields()) {
+			if (!StringUtil.equals(
+					ddmFormField.getType(),
+					DDMFormFieldTypeConstants.FIELDSET)) {
+
+				continue;
+			}
+
+			Map<String, DDMFormField> nestedDDMFormFieldsMap =
+				ddmFormField.getNestedDDMFormFieldsMap();
+
+			List<DDMFormField> orderedNestedDDMFormFields = new ArrayList<>();
```

Comment:
Hey @carolmariaabb,

1. I'm not sure if I follow the name of the method to rename this variable, like "reordered..."
2. I'm not sure if I should use "reorder vs order vs sort" to name the method and the variable :confused: 

What do you think?

Reply:
@marcelabc I would go with `sortNestedDDMFormFields` for the method name and `sortedNestedDDMFormFields` for the variable because it seems simpler